Tasks/VsTest/taskinputparser.ts

```diff
@@ -319,6 +337,51 @@ function getTiaConfiguration(): models.TiaConfiguration {
     return tiaConfiguration;
 }
 
+function getToolsInstallerConfiguration(): models.ToolsInstallerConfiguration {
+    const toolsInstallerConfiguration = {} as models.ToolsInstallerConfiguration;
+    
+    tl.debug("Path to VsTest from tools installer: " + tl.getVariable(constants.VsTestToolsInstaller.PathToVsTestToolVariable));
+    toolsInstallerConfiguration.vsTestPackageLocation = tl.getVariable(constants.VsTestToolsInstaller.PathToVsTestToolVariable);
+
+    // get path to vstest.console.exe
+    var matches = tl.findMatch(toolsInstallerConfiguration.vsTestPackageLocation, "**\\vstest.console.exe");
+    if (matches && matches.length !== 0) {
+        toolsInstallerConfiguration.vsTestConsolePathFromPackageLocation = matches[0];
+    } else {
+        utils.Helper.publishEventToCi(AreaCodes.TOOLSINSTALLERCACHENOTFOUND, tl.loc('toolsInstallerPathNotSet'), 1041, false);
+        throw new Error(tl.loc('toolsInstallerPathNotSet'));
+    }
+
+    // get path to Microsoft.IntelliTrace.ProfilerProxy.dll (amd64)
+    var amd64ProfilerProxy = tl.findMatch(toolsInstallerConfiguration.vsTestPackageLocation, "**\\amd64\\Microsoft.IntelliTrace.ProfilerProxy.dll");
+    if (amd64ProfilerProxy && amd64ProfilerProxy.length !== 0) {
+        toolsInstallerConfiguration.x64ProfilerProxyDLLLocation = amd64ProfilerProxy[0];
+    } else {
+        utils.Helper.publishEventToCi(AreaCodes.TOOLSINSTALLERCACHENOTFOUND, tl.loc('testImpactAndCCWontWork'), 1042, false);
+        tl.warning(tl.loc('testImpactAndCCWontWork'));
+    }
+
+    // Look in x64 also for Microsoft.IntelliTrace.ProfilerProxy.dll (x64)
+    amd64ProfilerProxy = tl.findMatch(toolsInstallerConfiguration.vsTestPackageLocation, "**\\amd64\\Microsoft.IntelliTrace.ProfilerProxy.dll");
+    if (amd64ProfilerProxy && amd64ProfilerProxy.length !== 0) {
+        toolsInstallerConfiguration.x64ProfilerProxyDLLLocation = amd64ProfilerProxy[0];
```

Comment:
this should go in else part of above code. if we don't find it in amd64 it will be there in x64 directoy. also rename
**\\amd64\\Microsoft.IntelliTrace.ProfilerProxy.dll
to
**\\x64\\Microsoft.IntelliTrace.ProfilerProxy.dll